src/NUnitEngine/nunit.engine/Internal/Logging/Logger.cs

```diff
@@ -165,9 +170,12 @@ private void WriteLog(InternalTraceLevel level, string message)
             writer.WriteLine(TRACE_FMT,
                 DateTime.Now.ToString(TIME_FMT),
                 level == InternalTraceLevel.Verbose ? "Debug" : level.ToString(),
-                System.Threading.Thread.CurrentThread.ManagedThreadId, name, message);
+#if !NETSTANDARD1_3
+                System.Threading.Thread.CurrentThread.ManagedThreadId,
```

Comment:
We could use `Environment.CurrentManagedThreadId` for all .NET Standard platforms. Note TRACE_FMT above needs updating in sync